### PR TITLE
Bump AWX version, after notable inventory deprecation

### PR DESCRIPTION
**Pull request by AlanCoding**
##### SUMMARY
New version

##### ISSUE TYPE
 - Feature Pull Request

##### AWX VERSION
```
13.0.0
```

##### ADDITIONAL INFORMATION
diff:

https://github.com/ansible/awx/compare/12.0.0...devel

I couldn't find much with observable effects on users, except for maybe #7372, which I don't know quite how I would articulate.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
 
+## 13.0.0 (Jun 19, 2020)
+- Removed scripts as a means of running inventory updates of built-in types (https://github.com/ansible/awx/pull/6911)
+- Ansible 2.8 is now unsupported. Some inventory source types are known to no longer work.
```
Comment (AlanCoding) [inline CHANGELOG.md]: might say "partially unsupported"

Comment (ryanpetrello) [reply CHANGELOG.md]: Yep, I think that's better.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
 
+## 13.0.0 (Jun 19, 2020)
```
Comment (ryanpetrello) [inline CHANGELOG.md]: Let's aim for maybe Monday on this, I know we're likely to merge @jrb's work tomorrow and it include it in this list.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
...
+- Removed scripts as a means of running inventory updates of built-in types (https://github.com/ansible/awx/pull/6911)
+- Ansible 2.8 is now unsupported. Some inventory source types are known to no longer work.
+- Fixed an issue where the vmware inventory source ssl_verify source variable was not recognized (https://github.com/ansible/awx/pull/7360)
```
Comment (jladdjr) [inline CHANGELOG.md:9]: 👍

**Review by jladdjr — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (jbradberry) [reply CHANGELOG.md]: I've updated along with mentioning import/export.

**Review by ryanpetrello — APPROVED** [review]

**Pull request merged**